client-app/src/desktop/tabs/examples/recalls/RecallsPanelModel.js

```diff
@@ -6,16 +6,16 @@
  */
 
 import {XH, HoistModel, LoadSupport} from '@xh/hoist/core';
+import {managed} from '@xh/hoist/core/mixins';
 import {GridModel} from '@xh/hoist/cmp/grid';
-import moment from 'moment';
 import {dateCol} from '@xh/hoist/cmp/grid/columns';
 import {compactDateRenderer} from '@xh/hoist/format';
-import {managed} from '@xh/hoist/core/mixins';
 import {Icon} from '@xh/hoist/icon/Icon';
 import {ONE_SECOND} from '@xh/hoist/utils/datetime';
 
 import {DetailsPanelModel} from './DetailsPanelModel';
 import {bindable} from '@xh/hoist/mobx';
+import moment from 'moment';
```

Comment:
is this import needed?

Reply:
Yupp-- we use moment in processRecord (new line 140).

Sorry for the confusion -- it looks like a new line when all I did was move it (see old line 10)